packages/server/tests/clients/servicesClient.ts

```diff
@@ -84,6 +84,21 @@ export default class ServicesClient {
     this.client.setOperator(AccountId.fromString(accountId), opPrivateKey);
   }
 
+  createPrivateKeyBasedOnFormat(operatorMainKey: string): PrivateKey {
```

Comment:
I believe this was also checked in before in hapuService, right? Could you pull it out and put in a neutral place like a util or something to share the method instead of duplicate it?